include/deal.II/base/bernstein_polynomial.h

```diff
@@ -0,0 +1,42 @@
+#include <deal.II/base/polynomial.h>
+#include <fstream>
+#include <iostream>
+
+#ifndef __deal2_bernstein_polynomial_h
+#define __deal2_bernstein_polynomial_h
+
+
+DEAL_II_NAMESPACE_OPEN
+
+/**
+ *  Class to compute the Bernstein basis polynomials of desire degree.
```

Comment:
What about "This class implements Bernstein basis polynomials" along with some further details, e.g., what they are used for and basis literature where to find a definition?